azure-pipelines.release-fluentui.yml

```diff
@@ -1,74 +1,223 @@
 pr: none
 trigger: none
 
-# Customize build number to include major version
-# Example: fluentui_20190626.1
-name: 'fluentui_$(Date:yyyyMMdd)$(Rev:.r)'
+schedules:
+  - cron: 0 3 * * *
+    branches:
+      include:
+        - master
+
+resources:
+  repositories:
+    - repository: self
+      type: git
+      ref: master
 
 variables:
   - group: 'Github and NPM secrets'
   - name: docsiteVersion
-    value: empty
+    value: 0.0.0-nightly
   - name: deployBasePath
-    value: empty
+    value: 0.0.0-nightly
   - name: officialRelease
-    value: 1
-
-pool: 'Self Host Ubuntu'
-
-steps:
-  - task: NodeTool@0
-    inputs:
-      versionSpec: '10.x'
-    displayName: 'Install Node.js'
-
-  - script: |
-      ver=`node -p "require('./packages/fluentui/react-northstar/package.json').version"`
-      echo "##vso[task.setvariable variable=deployBasePath]$ver"
-      echo "##vso[task.setvariable variable=docsiteVersion]$ver"
-      echo "Docsite base path published to for version $ver"
-    displayName: Set base path and version variables for job
-  - script: |
-      git config user.name "Fluent UI Build"
-      git config user.email "fluentui-internal@service.microsoft.com"
-      git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/microsoft/fluentui.git
-    displayName: Authenticate git for pushes
-  - task: Bash@3
-    inputs:
-      filePath: yarn-ci.sh
-    displayName: yarn
-  - script: |
-      PRODUCTION=true yarn build
-    displayName: yarn build
-  - script: |
-      NODE_ENV=production yarn build:fluentui:docs
-    displayName: yarn build:fluentui:docs
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish docsite as pipeline artifact'
-    inputs:
-      PathtoPublish: 'packages/fluentui/docs/dist'
-      ArtifactName: 'docsite_v$(docsiteVersion)'
-
-  - task: AzureUpload@2
-    displayName: Upload docs site
-    inputs:
-      SourcePath: 'packages/fluentui/docs/dist'
-      azureSubscription: 'UI Fabric (private)'
-      storage: fluentsite
-      ContainerName: '$web'
-      BlobPrefix: '$(deployBasePath)'
-      # uncomment to deploy to prod
-      # storage: fluentsite
-      # BlobPrefix: ''
-# No NPM release in CI for now
-#  - script: |
-#      yarn test
-#    displayName: yarn test
-#
-#  - script: |
-#      yarn lint
-#    displayName: yarn lint
-#
-#  - script: |
-#      yarn release:fluentui:$(bumpType)
-#    displayName: yarn release:fluentui:$(bumpType)
+    value: true
+  - name: nightlyRelease
+    value: true
```

Comment:
I would add all the pipeline variables here but commented, so we are aware hello of them